src/core/context/ActorContext.tsx

```diff
@@ -0,0 +1,16 @@
+import { useContext, createContext } from 'react';
+import type { EcosystemActor } from '../models/dto/teamsDTO';
+
+export type CoreUnitContextValues = {
```

Comment:
Note that this is an `AuditorContext` so the name should be according to that and avoid to use CoreUnit here

Reply:
you are right, done